README.md

```diff
@@ -35,7 +35,15 @@ Labels are assigned based on [Sensible Github Labels](https://github.com/Releque
     * links.json - JSON Hyper-Schema's Link Description Object meta-schema
     * hyper-schema-output.json - The recommended output format for JSON Hyper-Schema links
 
-Install "xml2rfc" using "pip" (https://pypi.org/project/xml2rfc/) and type "make" at a shell to build the .txt and .html spec files:
+The Makefile can create the necessary Python virtual environment for you:
+
+```sh
+make venv
```

Comment:
If you make `requirements.txt` a dependency of `.venv`, you can then make `.venv` a dependency of the default target. If so, users would only need to run `make` instead of `make venv` followed by `make`.

Given that and @webknjaz suggestion about the activation script, this could become just `make` as it was originally documented

Reply:
@handrews I think this comment still applies. The `venv` is now part of the `all` target, so users can just run `make`